docs/rules/core/0233-http-method.md

```diff
@@ -0,0 +1,64 @@
+---
+rule:
+  aip: 233
+  name: [core, '0233', http-method]
+  summary: Batch Create methods must use the POST HTTP verb.
+---
+
+# Batch Create methods: POST HTTP verb
+
+This rule enforces that all `BatchCreate` RPCs use the `POST` HTTP verb, as
+mandated in [AIP-233][].
+
+## Details
+
+This rule looks at any RPCs with the name beginning with `BatchCreate`, and
+complains if the HTTP verb is anything other than `POST`.
+
+## Examples
+
+**Incorrect** code for this rule:
+
+```proto
+// Incorrect.
+rpc BatchCreateBooks(BatchCreateBooksRequest) returns (BatchCreateBooksResponse) {
+  option (google.api.http) = {
+    get: "/v1/{parent=publishers/*}/books:batchCreate" // Should be `post:`.
```

Comment:
Use `put` as the bad example here (just because get + body makes no sense).

```suggestion
    put: "/v1/{parent=publishers/*}/books:batchCreate"  // Should be `post:`.
```